src/main/java/com/ftpix/sherdogparser/models/Fighter.java

```diff
@@ -10,7 +10,15 @@
     private String weight = "";
     private Date birthday;
     private int wins = 0;
+    private int winsKO = 0;
+    private int winsSUB = 0;
+    private int winsDEC = 0;
+    private int winsOTHER = 0;
     private int losses = 0;
+    private int loseKO = 0;
```

Comment:
Same comment as above about camel casing.
and  name of variable should be lossesKo, lossesSub lossesDec lossesOther to follow pattern set by wins (plural)